polaris-server.yml

```diff
@@ -167,3 +167,5 @@ logging:
       archivedLogFilenamePattern: ./logs/polaris-%d.log.gz
       # The maximum number of log files to archive.
       archivedFileCount: 14
+
+maxDocumentBytes: 0 # Don't set a limit
```

Comment:
If there's any situation where we might want to set this on a per-entity (catalog, namespace, table) level in the future I think this could be a PolarisConfiguration, otherwise I think this is good as-is

Reply:
From a DoS perspective I would expect all entity types to be about the same order of magnitude of size, so at least from that perspective I think just having a single global limit makes the most sense.